content/working_with/official_plugins/Orchestration/terraform.md

```diff
@@ -97,6 +97,10 @@ This refers to a Terraform module.
       
         **required:** false.
 
+    * `tfvars`: The name of the .tfvars file located at the same directory as the blueprint.
```

Comment:
In the same directory as the blueprint or located inside the Terraform Archive?

Reply:
Maybe - located inside the Terraform Archive at the source_path input ?

Reply:
Right. Should be relative to source_path